Packs/CortexXDR/Playbooks/Cortex_XDR_-_Retrieve_File_v2.yml

```diff
@@ -0,0 +1,322 @@
+      script: '|||xdr-retrieve-file-details'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      action_id:
+        complex:
+          root: PaloAltoNetworksXDR.RetrievedFiles
+          accessor: action_id
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 375
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "5_6_#default#": 0.68
+    },
+    "paper": {
+      "dimensions": {
+        "height": 920,
+        "width": 635,
+        "x": 10,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: endpoint_ids
+  value: {}
+  required: true
+  description: A comma-separated list of endpoint IDs.
+  playbookInputQuery:
+- key: file_path
+  value: {}
+  required: false
+  description: "A comma-separated list of file paths in any platform. It can be used instead of the macOS/Windows/Linux file paths. \nThe order of the files path list must be parallel to the endpoints list order. Therefore, the first file path in the list is related to the first endpoint ID and so on."
+  playbookInputQuery:
+- key: windows_file_paths
+  value: {}
+  required: false
+  description: |-
+    A comma-separated list of Windows paths.
+    Enter at least one path for either Windows, Linux, or Mac.
+  playbookInputQuery:
+- key: linux_file_paths
+  value: {}
+  required: false
+  description: |-
+    A comma-separated list Linux paths.
+    Enter at least one path for either Windows, Linux, or Mac.
+  playbookInputQuery:
+- key: mac_file_paths
+  value: {}
+  required: false
+  description: |-
+    A comma-separated list of Mac paths.
+    Enter at least one path for either Windows, Linux, or Mac.
+  playbookInputQuery:
+outputs:
+- contextPath: File
+  description: Retrieve the file details command results.
+- contextPath: File.Name
+  description: The full file name (including the file extension).
+  type: String
+- contextPath: File.EntryID
+  description: The ID for locating the file in the War Room.
+  type: String
+- contextPath: File.Size
+  description: The size of the file in bytes.
+  type: Number
+- contextPath: File.MD5
+  description: The MD5 hash of the file.
+  type: String
+- contextPath: File.SHA1
+  description: The SHA1 hash of the file.
+  type: String
+- contextPath: File.SHA256
+  description: The SHA256 hash of the file.
+  type: String
+- contextPath: File.SHA512
+  description: The SHA512 hash of the file.
+  type: String
+- contextPath: File.Extension
+  description: The file extension. For example, 'xls'.
+  type: String
+- contextPath: File.Type
+  description: The file type, as determined by libmagic (same as displayed in the file entries).
+  type: String
+tests:
```

Comment:
no need for all those tests - just check if any of those used the old one so you can change it accordingly.